ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Cleaner.java

```diff
@@ -412,7 +421,11 @@ private boolean removeFiles(String location, ValidWriteIdList writeIdList, Compa
     }
     StringBuilder extraDebugInfo = new StringBuilder("[").append(obsoleteDirs.stream()
         .map(Path::getName).collect(Collectors.joining(",")));
-    return remove(location, ci, obsoleteDirs, true, fs, extraDebugInfo);
+    boolean success = remove(location, ci, obsoleteDirs, true, fs, extraDebugInfo);
+    if (dir.getObsolete().size() > 0) {
+      updateDeltaFilesMetrics(ci.dbname, ci.tableName, ci.partName, obsoleteDirs);
```

Comment:
I regret suggesting that we include aborted directories in the obsolete count. 
1. There are other metrics about aborted directories.
2. previouslyActiveDeltas - (obsolete + aborted) != currentlyActiveDeltas, so the active delta count would be off.

My bad :/

Reply:
Reverted.